lib/api/multipartDelete.js

```diff
@@ -148,6 +151,8 @@ function multipartDelete(authInfo, request, log, callback) {
                         keys: [objectKey],
                         byteLength: partSizeSum,
                     });
+                    monitoring.promMetrics('DELETE', bucketName, '200',
+                        'abortMultipartUpload', partSizeSum);
```

Comment:
same here, don't you want to do that in line 154 callback?

Reply:
added

Reply:
added